uniffi_bindgen/src/bindings/python/templates/macros.py

```diff
@@ -5,19 +5,31 @@
 #}
 
 {%- macro to_ffi_call(func) -%}
-_UniFFILib.{{ func.ffi_func().name() }}({% call _arg_list_ffi_call(func.arguments()) -%})
+{%- match func.throws() -%}
+{%- when Some with (e) -%}
+_RustErrorHelper.try_raise(_RustErrorHelperPartial(_UniFFILib.{{ func.ffi_func().name() }},{% call _arg_list_ffi_call(func) -%}))
+{%- else -%}
+_UniFFILib.{{ func.ffi_func().name() }}({% call _arg_list_ffi_call(func) -%})
+{%- endmatch -%}
 {%- endmacro -%}
 
 {%- macro to_ffi_call_with_prefix(prefix, func) -%}
+{%- match func.throws() -%}
+{%- when Some with (e) -%}
+_RustErrorHelper.try_raise(_RustErrorHelperPartial(
```

Comment:
How can I test this issued code?
It's probably working, but I haven't tested it

Reply:
Hmmm Ya this would run when we have an `interface`. You can take a look at how the `sprites` example defines an interface if you wanna test this!